changelogs/fragments/42__eco-vcenter-ci-integration.yml

```diff
@@ -0,0 +1,4 @@
+---
+major_changes:
```

Comment:
this is not a major change- please change it 
major_changes->trivial

https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html

Reply:
done